cypress/integration/changeSignalAfterSubmit.spec.js

```diff
@@ -132,6 +132,9 @@ describe('Change signal after submit', () => {
       cy.wait(1000);
       cy.get(CHANGE_LOCATION.buttonSubmit).click();
 
+      // Check if background colour of changed element is flashing orange
+      createSignal.checkFlashingOrange();
```

Comment:
It's actually yellow

Reply:
Wow, difficult to see :-) But you are right.